### ISSUE TITLE
adonis-upgrade-kit@unknown can't be resolved to a satisfying range

### ISSUE THREAD

**Issue by Torvaldi** [adonisjs/upgrade-kit#1]
### Package version

1.3.0

### Describe the bug

While I'm trying to upgrade adonis v5 to v6 and after installing @adonisjs/upgrade-kit gloabally using yarn, this error message is visible and during the upgrade-packages script :
```sh
❯ yarn dlx adonis-upgrade-kit upgrade-packages
➤ YN0027: adonis-upgrade-kit@unknown can't be resolved to a satisfying range
➤ YN0035: The remote server failed to provide the requested resource
➤ YN0035:   Response Code: 404 (Not Found)
➤ YN0035:   Request Method: GET
➤ YN0035:   Request URL: https://registry.yarnpkg.com/adonis-upgrade-kit

➤ Errors happened when preparing the environment required to run this command.
```

Maybe I missed somthing, but I basically follow the Adonis Migration Guide (https://v6-migration.adonisjs.com/guides/upgrade-kit).

**Comment by Julien-R44** [comment]
I am not familiar at all with yarn and not sure of what is the equivalent of `npx` 

After reading this : https://stackoverflow.com/questions/66496166/is-there-any-yarn-equivalent-of-npx-preact-create I am thinking of removing instructions for Yarn and just keep npm and pnpm. Any suggestions ?

**Comment by RomainLanz** [comment]
> Any suggestions ?

I believe we can remove `yarn` specific command.
In the end, people not using the default should know what is the equivalent, we could even remove `pnpm` for that reason.

**Comment by Julien-R44** [comment]
yup thats what i did a few weeks ago. i removed all reference to yarn. we can close
